### DISCUSSION TITLE
이찬 - 헬스케어 - 수면장애 원인 분석 및 장애 여부 예측

### DISCUSSION THREAD

**Discussion by statschan2** [dinnercoding/wanted-pre-onboarding-challenge-data#109] [2023년 11월 프리온보딩 데이터 챌린지]
## 도메인 선택 이유
- 최근 취업 준비 과정에서 과도한 스트레스로 인하여 불면증을 겪고 있습니다. 
- 수면장애의 원인을 파악하여 생활에 적용할 수 있는 인사이트를 도출하고, 수면장애 여부를 예측하는 모델을 개발하여 자가진단 후 병원에 가는 것이 좋은지 판단할 수 있도록 해당 주제를 선택했습니다.

## 구체적인 문제 정의
- 수면장애 원인 파악
- 수면장애 여부 예측 모델 개발

## 데이터 업로드
- 데이터세트명 : Sleep Health and Lifestyle Dataset
- 출처 : Kaggle
- 링크 : https://www.kaggle.com/datasets/uom190346a/sleep-health-and-lifestyle-dataset/data

**Comment by ghost** [comment]
이찬님! 스트레스가 이만저만 많으신가 보네요 ㅠㅠ 자기계발도 중요하지만 자기 자신을 돌보는 것도 참 중요해요! 스트레스를 해소하는 자신만의 방법이 있으면 좋을 것 같아요. 저의 경우엔 극심한 스트레스를 받는 상황에서 감정을 스스로 정리해보려고 일기를 써보기도 한답니다. 통제할 수 있는 것과 통제할 수 없는 것을 정리한 후에 통제할 수 없는 것은 생각하지 않으려 하고 통제할 수 있는 것에 집중하려고 노력합니다.

이번 기회에 수면 장애와 관련한 여러 요인들을 살펴보시면 좋을 것 같고 자가진단을 하신다는 것은 참신한 발상이네요..ㅋㅋ
(힘드시다면 자가진단 전에 병원에 진료 받아보는 것이 어떨까 합니다..ㅎ) 모쪼록 몸조리 잘하시면 좋겠네요!
